docs/source/technical_tutorials/access/copc_access.ipynb

```diff
@@ -0,0 +1,291 @@
+{
```

Comment:
<p>There appears to be a missing step. I don't see where you converted the las file to a laz file. You have downloaded the las file above, but below the laz file has magically appeared without any conversion step.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/459/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>